raiden/tests/unit/test_cli.py

```diff
@@ -33,40 +16,6 @@ def cli_runner(tmp_path):
         yield partial(runner.invoke, env={"HOME": str(tmp_path)})
 
 
-def pytest_generate_tests(metafunc):
```

Comment:
Looks like this was accidentally removed.

Reply:
Not really, when i saw the failure in this test, which was pretty much checking the use of `--use-monitoring` or `--pfs-address` as parameters and then check to see if we are enabling a transport  that is **not** the matrix one. Since the UDP transport was removed, any option that requires the matrix will actually get it since it's the default and only option. I opted for removing the test because of the above.

Reply:
One more reason that is that i was not able to figure out how "successful" cases should be done. It seemed to me that the test expects as many `error messages` as `args`... but with i must have been doing something wrong... that kinda also influenced my decision to remove the test altogether.

Reply:
Yes, most of the option dependencies are transport specific but that doesn't mean we should remove the test for the whole feature IMO.

For example there have been new dependencies added for the routing mode (which don't seem to be tested yet):
https://github.com/raiden-network/raiden/blob/fd4d46794488504f3be533818305307109dfe1e8/raiden/ui/cli.py#L47-L50

> One more reason that is that i was not able to figure out how "successful" cases should be done

It doesn't ;) 
This only tests the failing cases.